From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Fri, 12 Jul 2013 21:29:19 -0400
Subject: [PATCH] Entity Serialization

add method to spawn an entity at a location if its not in any world (invalid)

squash entity serialization

diff --git a/src/main/java/com/domnian/paperdragon/api/DAPI_Entity.java b/src/main/java/com/domnian/paperdragon/api/DAPI_Entity.java
index 26609a7bd33caf380542d81b7ec23694eefb4312..30316ddecd8351d04d402b748cbd3472203d9a38 100644
--- a/src/main/java/com/domnian/paperdragon/api/DAPI_Entity.java
+++ b/src/main/java/com/domnian/paperdragon/api/DAPI_Entity.java
@@ -23,5 +23,16 @@
 
 package com.domnian.paperdragon.api;
 
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 public interface DAPI_Entity {
+
+    @Deprecated(since="1.18.2", forRemoval=true)
+    byte[] serializeEntity(@NotNull Entity craftentity);
+    @Deprecated(since="1.18.2", forRemoval=true)
+    @Nullable Entity deserializeEntity(byte[] data, @NotNull World world);
+
 }
